### PR TITLE
Add heatwave total

### PR DESCRIPTION
**Pull request by aulemahal**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated

### After merging to master and before closing the branch:
- [ ] bumpversion (minor / major / patch) has been called on `master` branch
- [ ] Tags have been pushed

* **What kind of change does this PR introduce?** 
Addition of a new indice to compute the *total* length of heat waves in a period. 

* **Does this PR introduce a breaking change?** 
No.

* **Other information**:
Has the same _standard\_name_ as `heat_wave_max_length`, but they both aren't exactly that anyway.

### CODE REVIEW

File: xclim/indices/_multivariate.py [hunk 1]
```diff
@@ -479,6 +481,62 @@ def heat_wave_max_length(
...
+    -----
+    See notes and references of `heat_wave_max_length`
+    """
+    freq = freq or "YS"
+    thresh_tasmax = utils.convert_units_to(thresh_tasmax, tasmax)
+    thresh_tasmin = utils.convert_units_to(thresh_tasmin, tasmin)
+
+    cond = (tasmin > thresh_tasmin) & (tasmax > thresh_tasmax)
+    group = cond.resample(time=freq)
+    total_count = group.apply(lambda d, dim=None: rl.windowed_run_count(d, window, dim=dim), dim="time")
```
Comment (huard) [inline xclim/indices/_multivariate.py]: Why are you passing `dim=None` ?

I think `group.apply(rl.windowed_run_count, window=window, dim="time")` would work as well.

Comment (tlogan2000) [reply xclim/indices/_multivariate.py]: This was my same question...  the apply call doesn't seem as 'clean' as heat wave max length etc.  Are there advantages to using a lambda func here?

Comment (aulemahal) [reply xclim/indices/_multivariate.py]: Yeah... I wasn't sure how to do it, I think my new commit fixes it.

File: xclim/indices/_multivariate.py [hunk 2]
```diff
@@ -533,7 +533,7 @@ def heat_wave_total_length(
 
     cond = (tasmin > thresh_tasmin) & (tasmax > thresh_tasmax)
     group = cond.resample(time=freq)
-    total_count = group.apply(lambda d, dim=None: rl.windowed_run_count(d, window, dim=dim), dim="time")
+    total_count = group.apply(lambda d: rl.windowed_run_count(d, window, dim='time'))
```
Comment (huard) [inline xclim/indices/_multivariate.py]: You can avoid the lambda altogether. Apply passes keyword arguments to the function.

File: xclim/indices/_multivariate.py (hunk 1 above)
Comment (tlogan2000) [reply xclim/indices/_multivariate.py]: I thin in this case we could simply do:

```return group.apply(rl.windowed_run_count, window=window, dim="time")```

File: xclim/indices/_multivariate.py (hunk 2 above)
Comment (tlogan2000) [reply xclim/indices/_multivariate.py]: Whoops commented in wrong spot:
https://github.com/Ouranosinc/xclim/pull/293#discussion_r345916778

Comment (aulemahal) [reply xclim/indices/_multivariate.py]: Ok! Did that. Sorry, I'm still not a complete xarray master.

Comment (tlogan2000) [reply xclim/indices/_multivariate.py]: No worries, me neither for that matter... there's a lot of stuff in there,

**Comment by tlogan2000** [comment]
@Zeitsperre Travis builds are failing on this PR .. any insights?

**Comment by Zeitsperre** [comment]
Bottleneck pushed a new version last night (1.3.0). Seems that it demands a specific version of numpy that doesn't agree with what is installed by default. I get the impression that this is bad package and I'm thinking we wait a day to see what's happening before stepping in. If this continues, the fix can be implemented here by pinning Bottleneck at 1.2.1.

**Comment by huard** [comment]
Should we bump the version to 0.12.0 with this ?

**Pull request merged**